### PR TITLE
Check blockchain connection during start process

### PR DESCRIPTION
**Pull request by marcelomorgado**
### Issue link

#95 #109

### Types of changes

Technical debt (a code change that doesn't fix a bug or add a feature but makes something clearer for devs)

### CODE REVIEW

**Comment by pcowgill** [comment]
@marcelomorgado Small merge conflict here after merging the other PR. Thanks!

File: demo/starter.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/* eslint no-console: "off" */
+const { exec } = require("child_process");
+const { Action } = require("tasit-sdk");
+const { ConfigLoader } = Action;
+const { ProviderFactory } = require("tasit-action/dist/ProviderFactory");
+const tasitSdkConfig = require("./config/default.js");
+
+const CONSOLE_FG_RED = "\x1b[31m";
+const CONSOLE_RESET = "\x1b[0m";
```
Comment (pcowgill) [inline demo/starter.js]: What are these strings?

Comment (marcelomorgado) [reply demo/starter.js]: These strings are used to change `console.log` color output (https://stackoverflow.com/a/41407246)

Comment (pcowgill) [reply demo/starter.js]: Cool. Thanks!

**Pull request merged**